cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -333,12 +336,6 @@ public TwillPreparer withConfiguration(Map<String, String> config) {
     if (config.containsKey(KubeMasterEnvironment.NAMESPACE_PROPERTY)) {
       programRuntimeNamespace = config.get(KubeMasterEnvironment.NAMESPACE_PROPERTY);
     }
-    if (config.containsKey(KubeMasterEnvironment.NAMESPACE_CPU_LIMIT_PROPERTY)) {
```

Comment:
shouldn't these still be used over the cConf values? Or are these supposed to be CPU and memory quotas for the namespace and not cpu/memory multipliers?

Reply:
If they're for namespace quota, we should rename them.

Reply:
These ones were applied at pod level which is not right since these are namespace level quotas.